bindata/network/kuryr/005-controller.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: kuryr-controller
+  namespace: openshift-kuryr
+  annotations:
+    kubernetes.io/description: |
+      This deployment launches the kuryr-controller component.
+spec:
+  template:
+    metadata:
+      name: kuryr-controller
+      labels:
+        app: kuryr
+        component: network
+        type: infra
+        openshift.io/component: network
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      serviceAccountName: kuryr
+      hostNetwork: true
+      priorityClassName: system-cluster-critical
+      containers:
+      - image: {{ .ControllerImage }}
+        name: controller
+{{ if (default true .ControllerEnableProbes) eq "true" }}
+        readinessProbe:
+          httpGet:
+            path: /ready
+            port: {{ default 8082 .ControllerProbesPort }}
+            scheme: HTTP
+          timeoutSeconds: 5
+        livenessProbe:
+          httpGet:
+            path: /alive
+            port: {{ default 8082 .ControllerProbesPort }}
+          initialDelaySeconds: 15
+{{ end }}
+        # FIXME(dulek): This shouldn't be required, but without it selinux is
```

Comment:
Weird. Maybe you can fix this by just symlinking `/etc/kuryr` to somewhere else in the Dockerfile?

Reply:
I just removed it, we'll deal with it if we'll encounter it again.